examples/bigcommerce/next-i18next.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  i18n: {
+    defaultLocale: 'en-US',
+    locales: ['en-US', 'es'],
+  },
+  localePath:
```

Comment:
Did this fix the issue???

Reply:
That's awesome! So glad it was a quick fix!

Reply:
Me too. I will create a PR for this in the POC tomorrow!